converters/toZigbee.js

```diff
@@ -1674,6 +1674,21 @@ const converters = {
             );
         },
     },
+    RM01_on_off: {
+        key: ['state'],
+        convertSet: async (entity, key, value, meta) => {
+            if (utils.hasEndpoints(meta.device, [0x12])) {
+                const endpoint = meta.device.getEndpoint(0x12);
+                return await converters.on_off.convertSet(endpoint, key, value, meta);
+            }
```

Comment:
Can we throw an exception in the else case? (otherwise it might be unclear for other users what goes wrong).

Reply:
Yes, I believe we can. I've updated the PR accordingly.